templates/bitcoin-sample.conf

```diff
@@ -13,20 +13,21 @@ rest=1
 # Low bandwidth/CPU optimizations
 blocksonly=1
 maxconnections=8
-dbcache=1000
+dbcache=<dbcache>
 maxmempool=300
 maxuploadtarget=5000
 
 # uncomment test if testnet=1 is activated
 #[test]
 # uncomment regtest if regtest=1 is activated
 #[regtest]
-port=8333
-rpcport=8332
+port=<port>
+rpcport=<port>
```

Comment:
Oh, I feel like `rpcport=<port>` is more consistent with everything else? Not sure how to explain it lol, but the line of thought was to enclose the "type" instead of the "what" inside the angled-brackets. Eg:

`bitcoind.rpcpass=<password>` (instead of `bitcoind.rpcpass=<rpcpass>`0
`tor.password=<password>` (instead of `tor.password<torpassword>`)

Reply:
Ahhh I get you, yeah that's fine then.